components/brave_shields/adblock/rs/src/convert.rs

```diff
@@ -12,8 +12,7 @@ use crate::ffi::{
 use adblock::blocker::BlockerResult as InnerBlockerResult;
 use adblock::lists::{ExpiresInterval, FilterListMetadata as InnerFilterListMetadata};
 use adblock::regex_manager::{
-    RegexDebugEntry as InnerRegexDebugEntry,
-    RegexDebugInfo as InnerRegexDebugInfo,
+    RegexDebugEntry as InnerRegexDebugEntry, RegexDebugInfo as InnerRegexDebugInfo,
```

Comment:
Are these format changes necessary? Seem unrelated to the rest of the PR.

Reply:
yah it does seem odd. since nothing else touches this file. Ill try to revert this and see what it does.